test/test/mocks/digital-exchange/components.js

```diff
@@ -0,0 +1,46 @@
EIF0GiIZzk3qbiEYepcr/Cn4kDpOAc7PJ4oUS/dq8HPRgo65LiVd+fO9dAPhuIBizUJnBjLIkCWQa0llwogAQnpaby8J4qHsrMg8RyfpZPKPB0a8QX+FRzQMs5faJ/vLi3CQqzGHRQv/tJV2UujZrOTb7l6a6TkyqWfzZWG1KlMqlPO7nWrub3JN0N8hfMcVNUwbEOE4ZjleFYCoEhMlpmCT2XL1FXKMKniNx3YfLp83BXk/UfNAxhCIcKa4dhvemiG6Bdo6sKtmiM6xi29N0vLGGoBSZjqnAMzCR3pV6ZeCxFywsXtOG+YRDFgBZs6YFvizBo5ILTK2OTTY/Iis8oiQOUfxxUna1El+GdRrwadljzVycOzD2hIno7UbnSbHHzTAU0cSaahEhtTDQeP91SOpr6MaSMfSm+Jf8ApdcQcB8CDD07UnEFduSOgOOsNmg6outBs4GzXaGmDYbHCKF5Hz/xQQwDEa9JKCM5GpVCXnvu6VDG421xS5RS25C1PmpMrnDWOB+80TJFeBNi23rTe4gynPNdEc4PFI4YoGzoPk8nim+6hmCcBtxHavFObhJl1H2caXqZmWDwg4+Wl1TTMyIw30N+BLakTqWSDCNJkUmxgOeSTKq5Z1zk6tWBRptwE1L3EF7ghYBFweAMnulFKABRWFQ8rdfKlIQlJEL54+Huw3kKu51xPz/KlY8JJLhG14y76xTA3CBkeAb7Abpk6EUCGVbFlVbRO6G4tbpJIt5ANndrULrleTfu+XsNiVn4uUloYYOPYSx12OuB4Z7ZplIJJe63YOK+LOKyaSMFqAM/tFJgUAZqnrZ11UqZkBT3znB73qc8YNKMJwjqhM1NeFEbW6hpuYJTD8wZ75YLMlqBcFoWdIYSjqxwEUthqg4M8AjBbKC/DOtGnSCTr8vtGu2BdVmIqyeRJD0hp4QJO7Wx9rI3GkQLwAiZDxkFjENWuCQkh6mv2KO0QMwEIEcRsLYWb3QHBEz5QD3vGXZKC4iwBKugv2uSM0qK+DSnlD721V3QgCAOL/WmGkJNbKLA30fk5GhL4zRRmE0jk1QGsGJGQeCQTQ5qZhzPW0NJsotood8E2OP26xYO2YLbGxDi9r0meJgVTfB48NmlpFlosJsNDCa7Q05s2etdSrgN12XS9XgkGVqotQpXJzArM9AfdI5gzlhoDjgKmd6Bun6DVN+aNxGOsXLwip0KXZBnPKTY08tqwZ0s12/RjJvUGiuSHLP3tgpIaWhbgD6Penz1y1ULovAsQXCDaABBc0DF2TIN4bSZTjxJInL/ANXJ4q9q8D4Mc2J+ijfJjkFm0WuiRdAEvV6CMljMxoD9VvbTPVDRdBMDN1amG+Xbq9OreuaKMJcCLg1IeDdBQhqmbtmt5vpBNPOeBocDmzGEHNE8mXEq2lRdxMt2mcYvAgWerB1v1rW3nds232Jp2gPgzl+IapIX2RLzKd9KvoOAljo67lB7f+VMMfpFWRbA4b6bEmCymHNNiVDxLqfBw1FCcCUb4R3LtCxzHCE8RnaU21K2+5vNgLpzPJRCKys3YS0HVrbxTd2pkMOwC0hwRhipq7eewdn6PYDxw7W/Z2ntQU6e4LJ2iNZvJiGIuz4Nn/Sjbg6Qtu1xph6sRlJMCSzbqoQG2kXH5wzMPNCtGWIQxIyFlF7olIQsnOQhMsxwkeIndP0UhLJyhzGcB1oEgGISBq/HGQ2epYaHGo+n2eo0Ng8CK4Jskk7sXp/PZQT2Bx9rOwsEutoaeT/Kd8CNvl9JfYo0yO5d4P8AbU8NF19UFjZwOPFxpSXQhAME1zHWhlVl5X0s0YMJUQ2vi9qASmRD8HLTAUrfUNt5wPtPCHJJPjlduWgJzCxERNPYrdTGzdyc5Y6sdL3rvFOi/IfPZS1Ql2ng7cYKclopYhvO7WLZZlgzFTgV5CZXw5OAMiUxLwzDw4eTLUkgSEDSXWbPlkqONsk21cHDLoKASOslhy4E8AURE3GSEgNWLt1DxUvZlG1PIvj34pCFMYAMj/XFACLsEJlF5dfPFCFUIJyG8ILy5G9hYKAxa3RNawnaTgz4HZguEYeFNlFGJL9FGRVHKqItWklIfLT4rIvs2vzzQI7qKA8PJSdAAytN9qxYiVjmKUr4I64qWwycKmQciVCi6uY8KC6TttPbilZoO4WzaefDV4XBwksLgJJ39sJAVJGW9ZJzehCwMVG+mEG5aeFi2aDSpN2OiCYms8IZITFsuzE87IinERjiUicyaGMUKgElwLBmILcYmihH+CcnFz2Wqci3QAhhMIeEqKEli7Nx2q5b4NopBIFG6utKGfWUDb9jK5Nr0EgBNlEIZEQYXJW5NESdEf2GSIbTGchpOpUCnOUMWZMDrTaA9pUHHIGnKs9ZodSNCAYAMmnJUBSFulPqEPkzSATokPgH60ockTLvbHJs/wAkwySt8Ns1f/rYFSVujpfWMNiOJbGZgeFujqUSrxLI5niuh80ctY+ifdNTevktdg1adoAMEyfmmibG4Stc4aTeaisq23FrUlMVCFwWR2sVnXMJbLFQd3wSapjUqhcso9WCeYK6ZCNt05iWOKAMADR2UDYmgkSkmNESt2WtsqxZndLixavb0J8EiVDkfSnlq8WKEmwYPcGlegRyLNuPWgME04oplli16atp+ZNxyli3SiWMAWBBYMFMLVI2OCdV5yVw9KAMAUQtdLtNb1stCZ+KFZbRBLlivhf4rDZNG6FpPT0WC40e1CIAGCOGHFNbwEO4AxDnrThJQGEdFOakYbpMzqW+b1ygEXuZbwZ4KgcQOSY2HWgQkSyNlh2lXu0MoIpOEm3FD+bgIDs8tbGyeYGCeCWCk3DCLSSI8iaoCGDBajSiCCmMrBK8xakslomghF/FEEgECwAY7AB0U3QBABiGCGkv+vVwm3XdXK2+EkDHSWO9KQblj3Ce8KTwtMIogojiC1NA2TzUgl73b9aJQBwBAXpKJUlgbxd5fdpwLB8NpkniiBgAIHmnNf/Z',
```

Comment:
please don't use images for each record, we also need to test how it behaves if the value is either empty string or null.